src/main/scala/jp/co/septeni_original/k8sop/util/FileReader.scala

```diff
@@ -21,7 +21,7 @@ object FileReader extends LazyLogging {
       .toList
       .filter(_.isFile)
       .map { f =>
-        (f.getName, FileUtils.readFileToString(f, StandardCharset.UTF_8))
+        (f.getName, FileUtils.readFileToString(f, StandardCharsets.UTF_8))
```

Comment:
Somehow this had depended on com.nimbusds.jose.util.StandardCharset and it disappeared in this PR.

At least here It's enough to use java.nio.charset.StandardCharsets instead of depending on other lib.